tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/steps/PitclipseSteps.java

```diff
@@ -93,6 +94,10 @@ public void mutationsAre(DataTable tableOfMutations) {
     @When("the following mutation is selected")
     public void mutationIsSelected(DataTable tableOfMutations) {
         PitMutation mutation = mutationsFromExampleTable(tableOfMutations).get(0);
+        mutationIsSelected(mutation);
+    }
+
+    public void mutationIsSelected(PitMutation mutation) {
```

Comment:
This method changes the current selection so wouldn't `selectMutation` or even `selectMutationInMutationsViews` be a better name?

Reply:
That's the same name as the one in the original Cucumber tests. But I agree: the name is misleading. Actually, it double clicks on the element on the view :)
What about `doubleClickMutationInMutationsView`?

Reply:
`doubleClickMutationInMutationsView` would be perfect